setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = SpatialDE
+name = SpatialDE2
```

Comment:
technically this was meant as version 2.0 of SpatialDE, that's why we kept the name. Having a package name that differs from the repository name might be confusing for users, I would prefer to keep that as-is.

Reply:
Ah yes this was my failed attempt to have both versions in my computer, I'll remove it